README.md

```diff
@@ -73,7 +73,7 @@ See code for all available configurations.
 | PC Engines APU                    | `<nixos-hardware/pcengines/apu>`                   |
 | [Raspberry Pi 2][]                | `<nixos-hardware/raspberry-pi/2>`                  |
 | [Samsung Series 9 NP900X3C][]     | `<nixos-hardware/samsung/np900x3c>`                |
-| [Purism Librem 13v3][]            | `<nixos-hardware/purism/librem/13v3>`              |
+| [Purism Librem 13v3/15v3][]       | `<nixos-hardware/purism/librem/13v3>`              |
```

Comment:
I would still add a second file even if we just make it a symlink for now.
The reason is that it allows us to introduce two files in future without breaking compatibility.

Reply:
No problem. I was considering this approach, too. Pushed a commit to resolve this issue.